### PR TITLE
Overview page

### PR DESCRIPTION
**Pull request by basafilm**
# Description

Url /overview and the component story added for overview page.
This component depend on search messages and users which both will be done by other developers. When they are finished will be added to overview page. 

Fixes # (issue)

# How to test?
http://localhost:3000/overview
or 
storybook

# Checklist

- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/Overview/OverviewSearchMessagesBtn.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      type="submit"
+      disabled={isLoading}
+      onClick={!isLoading ? handleClick : null}
+    >
+      {isLoading ? 'Loading…' : 'Chats'}
+    </button>
+  );
+}
+
+export default OverviewSearchMessagesBtn;
```
Comment (benna100) [inline src/client/components/Overview/OverviewSearchMessagesBtn.js:33]: What is this? Lucy has done the autocomplete and the search button

File: src/client/components/Overview/Overview.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Overview.styles.css';
+import OverviewSearchMessagesBtn from './OverviewSearchMessagesBtn';
+
+function Overview() {
+  return (
+    <div className="overview">
+      <h3 className="chatTitle">Chats</h3>
+      <div className="btnAndProfile">
+        <OverviewSearchMessagesBtn />
```
Comment (benna100) [inline src/client/components/Overview/Overview.js:10]: Have you talked with Lucy about implementing her component here? 

Also tariq's feature should go here! Remember to communicate :)

File: src/client/components/Overview/Overview.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Overview.styles.css';
+import OverviewSearchMessagesBtn from './OverviewSearchMessagesBtn';
+
+function Overview() {
+  return (
+    <div className="overview">
+      <h3 className="chatTitle">Chats</h3>
+      <div className="btnAndProfile">
```
Comment (benna100) [inline src/client/components/Overview/Overview.js:9]: btn -and-profile

Use this naming scheme for all classes, remember to update in css

File: src/client/components/Overview/Overview.stories.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Overview from './Overview';
+
+export default {
+  title: 'Overview',
+  component: Overview,
+};
+
+export const Component = () => <Overview title="chat-messages" />;
```
Comment (benna100) [inline src/client/components/Overview/Overview.stories.js:9]: This component has no props right!? SO what does this do?

File: src/client/components/Overview/Overview.styles.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  justify-content: space-between;
+  gap: 2em;
+}
+.chatTitle {
+  width: 90%;
+  height: 20%;
+  margin: 10px;
+  text-align: center;
+}
+.users {
```
Comment (benna100) [inline src/client/components/Overview/Overview.styles.css:16]: Its not your job to write the css for the components lucy and tariq did, they do that themselves

**Review by benna100 — CHANGES_REQUESTED** [review]
Looks pretty good Malek. There are a few misunderstandings it looks like. 

But linked Story! Only relevant commits! And the story was updated! You are improving, super happy to see that!

By the way i have set the priority to high on this task, this is an essential task for version 1.0 to work

File: src/client/components/Overview/OverviewSearchMessagesBtn.js (hunk 1 above)
Comment (basafilm) [reply src/client/components/Overview/OverviewSearchMessagesBtn.js:33]: It is a button for the Chats which located at the bottom of mockup 1. It is not working because it should be connected to the messages / chats.I don't know who is working on that.

File: src/client/components/Overview/Overview.js (hunk 2 above)
Comment (basafilm) [reply src/client/components/Overview/Overview.js]: I added lucys component and its working fine 
But Tariq's , UsrersList , is not completed . It is just a template and returning nothing. I talked with him says my work finished > I think I have to complete it and fetch the users list ?
Do I ?

File: src/client/components/Overview/Overview.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+          <Link to="/profile">Profile</Link>
+          <Switch>
+            <Route>
+              <Route exact path="/profile">
+                <Profile />
+              </Route>
+              <OverviewSearchMessagesBtn />
+            </Route>
+          </Switch>
+        </Router>
```
Comment (benna100) [inline src/client/components/Overview/Overview.js:27]: Malek you keep doing lots of things in this pr that is not part of the PR!

This is why we keep talking about communication and why it is so important. We have spend quite a lot of time on this pr because you misunderstood what you were to do and just started coding and now we are losing time here!

We have a file for the router and that should not be set up here.

File: src/client/App.js [hunk 7]
```diff
@@ -33,6 +34,9 @@ function App() {
           >
             <Profile />
           </AuthenticatedRoute>
+          <Route>
+            <Overview exact path="/overview" />
+          </Route>
```
Comment (benna100) [inline src/client/App.js:39]: This is where the profile routing should be set up. BUT it is not part of your task, so dont do it

**Review by benna100 — CHANGES_REQUESTED** [review]
Remember to understand the story before making any code

File: src/client/components/Overview/Overview.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+import Search from '../Search/Search';
+
+function Overview() {
+  return (
+    <div className="overview">
+      <h3 className="chat-title">Chats</h3>
+      <div className="search">
+        <Search />
+      </div>
+      <div className="users-list">{/* <UsersList /> */}</div>
```
Comment (benna100) [inline src/client/components/Overview/Overview.js:15]: Dont outcomment this, we need to see the UserList in order to create a new conversation

**Review by benna100 — CHANGES_REQUESTED** [review]

File: src/client/components/Overview/Overview.js (hunk 8 above)
Comment (panivita) [reply src/client/components/Overview/Overview.js:15]: I need it ASAP!!! For my ticket https://github.com/HackYourFuture-CPH/chattie/issues/38

Comment (benna100) [reply src/client/components/Overview/Overview.js:15]: Okay, we will merge. Then you can work on it victoria

**Review by benna100 — APPROVED** [review]
